### ISSUE TITLE
Coordinates in FullScreen Map

### ISSUE THREAD

**Issue by Vengos** [Nevcairiel/Mapster#43]
When i enlarge map to full screen i don't see coordinates anywhere.

**Comment by babelfish** [comment]
This issue still exists 2 years later.

**Comment by johnip86** [comment]
I was able to fix this pretty quickly. 
In the most recent version on curseforge I was having the same isssue.
On the small map that shows the quest log, the coordinates are below the map and outside the frame. It's the same case with the full screen map, but you can't see them because they're off the screen... 
Open the **Coords.lua** file and check lines 112 and 113. You should see this:

```
cursortext:SetPoint("TOPLEFT", WorldMapFrame.ScrollContainer, "BOTTOM", 30, WoWClassic and -9 or -5)
playertext:SetPoint("TOPRIGHT", WorldMapFrame.ScrollContainer, "BOTTOM", -30, WoWClassic and -9 or -5)
```

Change to this:
```	
cursortext:SetPoint("BOTTOMLEFT", WorldMapFrame.ScrollContainer, "BOTTOM", 30,5, WoWClassic and -9 or -5)
playertext:SetPoint("BOTTOMRIGHT", WorldMapFrame.ScrollContainer, "BOTTOM", -30,5, WoWClassic and -9 or -5)
```

If you don't add the 5 point offset in the Y dimension, they are stuck under the border and halfway off the screen on the full screen map.
@babelfish

**Comment by PirateKain** [comment]
Yes same here. The coordinates needs to either move to the top map bar or in with the zone dropdowns on the right side. Or add positioning options so we can move it to where we'd like it best ourselves, like positioning sliders.
